--- carddav.py	2016-01-15 09:34:25.185529251 +0100
+++ carddav.py	2016-01-15 09:29:47.232198623 +0100
@@ -100,8 +100,10 @@
                                         headers=headers,
                                         **self._settings)
         response.raise_for_status()   # raises error on not 2XX HTTP status code
+        """ removed to work with davical of dotsource
         if response.headers['DAV'].count('addressbook') == 0:
             raise Exception("URL is not a CardDAV resource")
+        """
 
     @property
     def verify(self):
